index.html

```diff
@@ -1169,19 +1169,66 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a> so it
+is independent of any specific representation. In this section we provide a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification such that the value of any property
+may be losslessly converted into another representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with this specification.
+        </li>
+      </ol>
     </section>
-
     <section>
-      <h2>Representation Requirements</h2>
+      <h2>Extensibility</h2>
+      <p>
+The data model supports two types of extensibility.
+      </p>
+      <ol>
+        <li>
+Registry-based extensions are RECOMMENDED for maximum interoperability. The use
+of the registry for properties is the only guaranteed method for two independent
+implementations to be able to work together. Registry-based extensions MUST be
+losslessly convertible between all conformant representations. The official W3C
```

Comment:
Should we define what we mean by losslessly. I assume it can be inferred from the standard definition, but figured the questions was worth getting other opinions on this.

Reply:
My rationale was that "lossless" is a well-known concept in digital tech. Wikipedia doesn't have a definition of the word by itself but does have https://en.wikipedia.org/wiki/Lossless_compression which starts:

> Lossless compression is a class of data compression algorithms that allows the original data to be perfectly reconstructed from the compressed data.

So my sense is that we don't have to provide a definition, however I'm fine with doing that if others think it would clarify the spec.

Reply:
Yeah, I understood what you meant to just didn't want it to get captured as tribal knowledge was the reason I was checking.

Reply:
We can define "lossless" in the context of our specification. What we mean is what @talltree said -- you can get data in and out of a representation without losing anything from that representation. That means anything you can represent in the DID document model can be represented in any compliant representation, and you can't pull tricks like "this is a Dates In The Julian Calendar Type that only my system understands" without defining a general model and representation for that across different formats.

Reply:
I think I was the one who originally introduced the term "lossless" into the conversation. Before the F2F, one common argument was that a DID document could have a JSON "encoding" and a JSON-LD "encoding", but that argument quietly assumed that you would simply remove or add a `@context` (with that oversimplified approach however, the conversion is not guaranteed to be "lossless"). This is also why I preferred the term "representation" over "encoding".

Reply:
Made this change in commit [77a9787](https://github.com/w3c/did-core/pull/241/commits/77a97872959692d262c16bbd827f857677ba8f82) to add a definition of "lossless" plagiarized from @jricher's comment.